.github/workflows/budibase_ci.yml

```diff
@@ -239,6 +239,7 @@ jobs:
       - name: Test server
         env:
           DATASOURCE: ${{ matrix.datasource }}
+          DEBUG: testcontainers*
```

Comment:
Probably not needed

Reply:
I was getting timeouts in the oauth2 tests and they went away when I added this to debug them 🫠